tests/datasets/test_idtrees.py

```diff
@@ -19,6 +19,9 @@
 import torchgeo.datasets.utils
 from torchgeo.datasets import IDTReeS
 
+pytest.importorskip("pandas", minversion="0.19.1")
+pytest.importorskip("laspy", minversion="2")
```

Comment:
Some of the tests that don't use the `dataset` fixture still use `IDTReeS` and will fail if these deps aren't installed.